MemberExpectations.md

```diff
@@ -0,0 +1,58 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the code of conduct but that supports
+the vision, mission and operating principles of the Node.js project.
+
+While members have the right and responsibility to exercise independent
+judgment and to express dissenting opinions during discussions within
+the organization, members also have the obligation outside the
+organization (twitter, reddit, etc.) to respect and support decisions
+and values of the majority, even when a member dissented from
+the majority view. A member who does not support a team decision
+or current team value may express the member's opposition within
+the team in an appropriate manner, but must not take actions publicly
+that have the purpose or result of undermining the decisions, values
+or actions of the team. Accordingly, a member who itends to publicly
```

Comment:
I think this casts a *very* wide net, and we should be more specific as to what decisions we refer to. I don’t think anybody should resign because they had a different opinion on what we should do with the `Buffer` constructor, for example.

Reply:
I agree with @addaleax here.This is beginning to read like a general gag order on discourse, which I don't think is the exact intention.

Expressing disagreement when done in a respectful way that doesn't undermine the project should be possible.

I am also not super keen on the choice of the term `dissenting`. Here is an alternative pass to this paragraph, feel free to cherry pick bits you like or take it wholesale

```
Members are encourage to have individual opinions and to express those
opinions during discussions within the organization, members also have
the obligation outside of the organization (social media) to be respectful
of the decisions and values of the majority. A member who does not support
a team decision or current team value may express the member's opposition
within the team in an appropriate manner, but must be extremely careful
when expressing the opinion externally. External communication must not
be done with the intent of undermining a decision, project values, or
actions of the team. If another team member expresses that they believe
external communications appear to have the intent to undermine the committe
it is the responsibility of the individual expressing their opinions to rectify
the situation immedietely. If a member feels that there is no recourse aside
from going publicly regarding extremely contencious issue they should
resign from the team before doing so.
```

Reply:
Seems reasonable to me will incorporate in next pass.